paper/paper.md

```diff
@@ -0,0 +1,323 @@
+---
+title: 'Developing and deploying an integrated workshop curriculum teaching computational skills for reproducible research'
+tags:
+  - reproducible research
+  - data science
+  - R
+  - git
+  - Unix shell
+authors:
+  - name: Zena Lapp^[co-first author]
+    orcid: 0000-0003-4674-2176
+    affiliation: 1
+  - name: Kelly L. Sovacool^[co-first author]
+    orcid: 0000-0003-3283-829X
+    affiliation: 1  
+  - name: Nick Lesniak
+    orcid:
+    affiliation: 2
+  - name: Dana King
+    orcid:
+    affiliation: 3
+  - name: Catherine Barnier
+    orcid:
+    affiliation: 1
+  - name: Matthew Flickinger
+    orcid:
```

Comment:
My orcid is: 0000-0002-8528-8925

Reply:
@MrFlick what should we list as your affiliation?

Reply:
Department of Biostatistics, University of Michigan